src/main/resources/l10n/JabRef_de.properties

```diff
@@ -2448,17 +2448,17 @@ Shared_version\:_%0=Gemainsam_genutzte_Version\:_%0
 Please_merge_the_shared_entry_with_yours_and_press_"Merge_entries"_to_resolve_this_problem.=Bitte_führen_Sie_den_gemeinsam_genutzten_Eintrag_mit_Ihrem_zusammen_und_klicken_Sie_auf_"Einträge_zuammenführen",_um_das_Problem_zu_beheben.
 Canceling_this_operation_will_leave_your_changes_unsynchronized._Cancel_anyway?=Durch_einen_Abbruch_dieser_Operation_werden_die_Änderungen_nicht_synchronisiert._Trotzdem_abbrechen?
 The_BibEntry_you_currently_work_on_has_been_deleted_on_the_shared_side._Hit_"Keep"_to_recover_the_entry.=Der_Eintrag,_welchen_Sie_zur_Zeit_bearbeiten,_ist_auf_der_gemeinsam_genutzen_Datenbank_nicht_mehr_vorhanden._Klicken_Sie_auf_"Behalten",_um_den_Eintrag_wiederherzustellen.
-Cannot_cite_entries_without_BibTeX_keys._Generate_keys_now?=
+Cannot_cite_entries_without_BibTeX_keys._Generate_keys_now?=Kann_keine_Einträge_ohne_BibTeX-Key_zitieren._Keys_jetzt_generieren?
 New_technical_report=Neuer_technischer_Bericht
 
-%0_file=
-Custom_layout_file=
-Protected_terms_file=
-Style_file=
+%0_file=%0_-Datei
```

Comment:
In my German version of JabRef, I see translations like ArXiv-Spalte, BibTeX-Felder and BibTeX-Datei. To be consistent it should be PubMed-Datei and not PubMed -Datei.